libraries/csharp_dotnetcore/Microsoft.Bot.Builder.Adapters.WeChat/Microsoft.Bot.Builder.Adapters.WeChat.csproj

```diff
@@ -20,7 +20,7 @@
   </PropertyGroup>
   
   <ItemGroup>
-    <PackageReference Include="AdaptiveCards.Rendering.Html" Version="1.1.2" />
+    <PackageReference Include="AdaptiveCards.Rendering.Html" Version="1.2.3" />
```

Comment:
This version has a serious bug, use version 1.2.4.

Reply:
sames I pushed a wrong version, fixed